lib/http_requests.py

```diff
@@ -13,17 +13,13 @@
 
 
 def get_all_ocp_build_data_branches():
-
     """
     This function lists all the branches of the ocp-build-data repository.
     :return: dict, all the branches along with their details.
     """
 
-    access_token = os.environ["GITHUB_PERSONAL_ACCESS_TOKEN"]
-
     try:
-        req = requests.get(app_constants.GITHUB_URL_TO_LIST_ALL_OCP_BUILD_DATA_BRANCHES,
-                           headers={"Authorization": "token " + access_token})
```

Comment:
why remove auth?

yes, it's an unauthenticated endpoint, but i think this was intended to solve rate-limiting problems. our token has a way higher rate-limit than you get without it.